src/Generators/SonyflakeGenerator.php

```diff
@@ -11,10 +11,40 @@
  *
  * @see https://github.com/sony/sonyflake
  */
-class SonyflakeGenerator implements ISnowflakeGenerator
-{
-    public function generate(): int
-    {
-        // TODO: Implement generate() method.
-    }
+class SonyflakeGenerator implements ISnowflakeGenerator{
+
+	private int $microtimeLength;
+	private int $machineLength;
+	private int $sequenceLength;
+
+	public function __construct(){
+
+		$this->microtimeLength = 39;
+		$this->machineLength = 16;
+		$this->sequenceLength = 8;
```

Comment:
Same here, default values should not be set in the constructor, but at the variable declaration itself.